internal/pkg/validator/validator.go

```diff
@@ -56,19 +74,16 @@ func Validate(value interface{}, rules ...Validation) error {
 	return nil
 }
 
-func processValidateError(err validator.ValidationErrors) error {
+func processValidateError(err validator.ValidationErrors, translator ut.Translator, fieldName string) error {
 	result := utils.NewMultiError()
 	for _, e := range err {
-		// Remove struct name, first part
-		namespace := regexpcache.MustCompile(`^([^.]+\.)?(.*)$`).ReplaceAllString(e.Namespace(), `$2`)
-		// Hide nested fields
-		namespace = strings.ReplaceAll(namespace, `__nested__.`, ``)
-		result.Append(fmt.Errorf(
-			"key=\"%s\", value=\"%v\", failed \"%s\" validation",
-			namespace,
-			e.Value(),
-			e.ActualTag(),
-		))
+		if fieldName == "" {
+			// Remove struct name, first part
+			fieldName := regexpcache.MustCompile(`^([^.]+\.)?(.*)$`).ReplaceAllString(e.Namespace(), `$2`)
```

Comment:
Není mi úplně jasný jak se ten field name dostane do `err.Namespace()`, každopádně při validaci input Rules v něm nic není, tak sem odtamtud předávám `fieldName` ručně a teprve pokud je prázdný, tak se vezme z `.Namespace()`. Je to ok?

Reply:
Tady se to ale začíná komplikovat, protože hláška po překladu už obsahuje tu hodnotu z `err.Namespace()`, takže to původní odstraňování "struct name" a "hiding nested fields" nejde snadno udělat. Muselo by se to nějak vypreparovat z té přeložené hlášky a nahradit v ní a nebo ještě přetížit a upravit přímo ten překlad.